orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/AccountInterventionService.java

```diff
@@ -141,12 +143,16 @@ private AccountIntervention handleException(Exception e) {
                 configurationService.getAccountInterventionsErrorMetricName(),
                 Map.of("Environment", configurationService.getEnvironment()));
         if (accountInterventionsActionEnabled && acountInterventionsAbortOnError) {
+            LOG.error(
+                    "Problem communicating with Account Intervention Service. Aborting user journey",
+                    e);
             throw new AccountInterventionException(
-                    "Problem communicating with Account Intervention Service", e);
+                    "Problem communicating with Account Intervention Service. Aborting user journey",
+                    e);
         }
-        LOG.error(
-                "Problem communicating with Account Intervention Service. Assuming no intervention. ",
-                e);
+        LOG.atLevel(AIS_FAIL_OPEN)
+                .log(
+                        "Problem communicating with Account Intervention Service. Assuming no intervention");
```

Comment:
I'm not sure how I feel about this - it feels a bit unusual... could we create some kind of metric and alert on that?

Reply:
In fact is there a need for this? Can we not just log at error level, and then just use the flags you've already got above to decide if it's p1 & goes to pagerduty?

Reply:
Would still be nice to use a metric for this though, means we're not just alarming on all cases of error logs in this lambda

Reply:
> In fact is there a need for this? Can we not just log at error level, and then just use the flags you've already got above to decide if it's p1 & goes to pagerduty?

I don't think this can work - as long as there's logs at level ERROR in the processing identity lambda in prod we will trigger pager duty. It can't distinguish whether it's failing open or aborting on error.

Unless... we use a different sort of pattern matching for the AIS fail open metric, like matching a particular string in the logs. But this feels quite brittle (easy to accidentally change the log message and break it). The solution here seems more resilient and neater (to me anyway 🙂)